raiden/tests/utils/network.py

```diff
@@ -536,7 +542,7 @@ def wait_for_token_networks(
 
 def wait_for_channels(
     app_channels: AppChannels,
-    registry_address: PaymentNetworkAddress,
+    payment_network_address: PaymentNetworkAddress,
     token_addresses: List[TokenAddress],
     deposit: TokenAmount,
```

Comment:
Just for readability: Unless `deposit` is used consistently thorughout the code to indicate an amount , it would make `deposit_amount` would make it clear that we are to a `TokenAmount` and not to any instance of payment..